utils.sh

```diff
@@ -0,0 +1,242 @@
+  export CONJUR_APPLIANCE_URL="https://cuke-master"
+  docker-compose up -d client cuke-master test-dap
+
+  # Delay to allow time for conjur to come up
+  echo 'Waiting for conjur server to be healthy'
+  docker-compose run --rm test ./wait_for_server.sh
+}
+
+function loadDapTestPolicy() {
+  echo '-----------------------test.sh------------------------------'
+  echo "Loading DAP test policy"
+  echo '------------------------------------------------------------'
+
+  dap_client_cid=$(docker-compose ps -q client)
+
+  # Get certificate from cuke-master
+  ssl_cert=$(docker-compose exec -T cuke-master cat /opt/conjur/etc/ssl/conjur.pem)
+
+  docker exec \
+    -e CONJUR_SSL_CERTIFICATE="$ssl_cert" \
+    ${dap_client_cid} conjur authn login -u admin -p SEcret12!!!!
+
+  # copy test-policy into a /tmp/test-policy within the client container
+  docker cp test-policy ${dap_client_cid}:/tmp
+
+  docker exec \
+    -e CONJUR_SSL_CERTIFICATE="$ssl_cert" \
+    ${dap_client_cid} conjur policy load root /tmp/test-policy/root.yml
+}
+
+function initializeDapCert() {
+  echo '-----------------------test.sh------------------------------'
+  echo "Fetch certificate for DAP using client cli"
+  echo '------------------------------------------------------------'
+
+  dap_client_cid=$(docker-compose ps -q client)
+
+  # Get the pem file from conjur server
+  CONJUR_ACCOUNT="cucumber"
+  CONJUR_PROXY="https://cuke-master"
+
+  echo "remove old pem file"
+  rm -rf /test-cert/*
+
+  echo "fetch pem file from enterprise server"
+  exec_command="echo yes | conjur init -u '${CONJUR_PROXY}' -a '${CONJUR_ACCOUNT}'"
+  docker exec ${dap_client_cid} /bin/bash -c "$exec_command"
+
+  echo "convert pem to der file and copy it to share memory"
+  convert_command="openssl x509 \
+     -outform der \
+     -in /root/conjur-cucumber.pem \
+     -out /test-cert/conjur-cucumber.der"
+  docker exec ${dap_client_cid} ${convert_command}
+
+  echo "import cert inside DAP test container"
+  dap_test_cid=$(docker-compose ps -q test-dap)
+
+  # Import cert converted above into keystore
+  JAVA_PATH=$(docker exec ${dap_test_cid} sh -c 'echo $JAVA_HOME')
+  import_command="keytool \
+     -import \
+     -alias cuke-master -v \
+     -trustcacerts \
+     -noprompt \
+     -keystore "${JAVA_PATH}/jre/lib/security/cacerts" \
+     -file /test-cert/conjur-cucumber.der -storepass changeit"
+  docker exec ${dap_test_cid} ${import_command}
+}
+
+function runDapTests() {
+  echo '-----------------------test.sh------------------------------'
+  echo "Running DAP tests"
+  echo '------------------------------------------------------------'
+
+  dap_test_cid=$(docker-compose ps -q test-dap)
+
+  api_key_admin=$(docker-compose exec -T client conjur user rotate_api_key)
+
+  # Execute DAP tests
+  docker exec \
+  -e CONJUR_AUTHN_API_KEY=${api_key_admin} \
+  -e CONJUR_AUTHN_LOGIN="admin" \
+  ${dap_test_cid} \
+    mvn test
+}
+
+function createOssTestEnvironment() {
```

Comment:
Now that we're using this for building an interactive environment and for testing, what do you think about making this function name a bit more generic?  What about something like `createOssEnvironment`?

Reply:
Changed name